plugins/csp-wms-overlays/src/WebMapLayer.cpp

```diff
@@ -0,0 +1,185 @@
+////////////////////////////////////////////////////////////////////////////////////////////////////
+//                               This file is part of CosmoScout VR                               //
+//      and may be used under the terms of the MIT license. See the LICENSE file for details.     //
+//                        Copyright: (c) 2020 German Aerospace Center (DLR)                       //
+////////////////////////////////////////////////////////////////////////////////////////////////////
+
+#include "WebMapLayer.hpp"
+#include "logger.hpp"
+#include "utils.hpp"
+
+#include "../../../src/cs-utils/utils.hpp"
+
+#include <curlpp/Easy.hpp>
+#include <curlpp/Options.hpp>
+
+namespace csp::wmsoverlays {
+
+////////////////////////////////////////////////////////////////////////////////////////////////////
+
+WebMapLayer::WebMapLayer(VistaXML::TiXmlElement* element, Settings settings)
+    : mSettings(settings) {
```

Comment:
This should either be moved (if possible, I am not sure) or passed per `const&`.

```suggestion
    : mSettings(std::move(settings)) {
```